### DISCUSSION TITLE
New REACTIVE_POWER and REACTIVE_ENERGY SensorDeviceClasses

### DISCUSSION THREAD

**Discussion by balazs92117** [home-assistant/architecture#955] [Entity Models]
I have a sensor (dsmr - p1 port) that provides data with unit of `kvarh`. It is the same as POWER <-> ENERGY, so REACTIVE_ENERGY is the total reactive power over time. For this I would like to create a new SensorDeviceClass with the name of REACTIVE_ENERGY. It would have units of `varh` and `kvarh`.

This was also mentioned in an other discussion: https://github.com/home-assistant/architecture/discussions/688 which is already has an answer, but with limited 

I already created a PR for this: https://github.com/home-assistant/core/pull/98453 (the PR includes possible solution for  https://github.com/home-assistant/architecture/discussions/729 too)

**Comment by frenck** [comment]
This is two proposals in one. Please split those. Especially considering kvar was proposed here already (so a duplicate): <https://github.com/home-assistant/architecture/discussions/729>
